src/Native/SymSgdNative/CMakeLists.txt

```diff
@@ -18,4 +18,8 @@ endif()
 add_library(SymSgdNative SHARED ${SOURCES} ${RESOURCES})
 target_link_libraries(SymSgdNative PUBLIC ${MKL_LIBRARY})
 
+if(APPLE)
+    set_target_properties(SymSgdNative PROPERTIES INSTALL_RPATH "@loader_path;@loader_path/../../../../../${MKL_LIB_RPATH};${MKL_LIB_PATH}")
```

Comment:
`${MKL_LIB_PATH}` shouldn't be necessary, right? When we are running in our repo, these assemblies should live together, so `@loader_path` should just take care of it.